C/CompilerSupportLibraries/build_tarballs.jl

```diff
@@ -1,17 +1,36 @@
-using BinaryBuilder, SHA
+using BinaryBuilder, BinaryBuilderBase, SHA
 
 include("../../fancy_toys.jl")
 
 name = "CompilerSupportLibraries"
-version = v"0.6.1"
+version = v"0.5.2"
```

Comment:
Why downgrading the version number?

Reply:
Sorry, that was just for testing stince the original libstdxx jll build was for that version. I'll undo this.